src/main/java/com/internet/shop/model/Order.java

```diff
@@ -0,0 +1,30 @@
+package com.internet.shop.model;
+
+import java.util.List;
+
+public class Order {
+    private Long id;
+    private List<Product> products;
+    private Long userId;
+
+    public Order(List<Product> products, Long userId) {
+        this.products = products;
```

Comment:
we won't receive `products` from outside

Reply:
fixed